06-classification_continued.Rmd

```diff
@@ -236,9 +321,12 @@ different train / validation splits, we'll get a better estimate of accuracy,
 which will lead to a better choice of the number of neighbours $K$ for the
 overall set of training data. 
 
-> **Note:** you might be wondering why we can't we use the multiple splits to test our final classifier after tuning is done. This is simply
-> because at the end of the day, we will produce a single classifier using our overall training data. If we do multiple train / test splits, we will
-> end up with multiple classifiers, each with their own accuracy evaluated on different test data.
+> **Note:** you might be wondering why we can't we use the multiple splits to
```

Comment:
I wonder if this note comes too early? I think they at least need to see CV and the CV figure before they might be able to even think about this.

Reply:
moved it later